src/slave/containerizer/mesos/isolators/cgroups2/cgroups2.hpp

```diff
@@ -149,7 +149,7 @@ class Cgroups2IsolatorProcess : public MesosIsolatorProcess
 
   process::Future<Nothing> __cleanup(
       const ContainerID& containerId,
-      const std::vector<process::Future<Nothing>>& futures);
+      const process::Future<Nothing>& future);
```

Comment:
There was already a `__cleanup` function (that wasn't being used) inside of the `Cgroups2IsolatorProcess`. Therefore, rather than having to _add_ a `__cleanup` function, we just needed to update the parameters it took.